jobs/workflow_test.groovy

```diff
@@ -131,7 +129,6 @@ import utilities.StatusUpdater
 
     environmentVariables {
       env('COMMIT', isMaster)
-      env('PARALLEL_TESTS', true)
```

Comment:
Good question.  This was a temporary env var used when we had two e2e charts, one for running tests in a serialized fashion, the other concurrently.  We now just have one e2e chart, defaulting to running in parallel (using multiple `GINKGO_NODES`).  To run in serial using the same chart, we can set `GINKGO_NODES` to `1`.